src/app/page.tsx

```diff
@@ -1,12 +1,14 @@
 "use client";
 
 import ExampleButton from "@components/ExampleButton";
+import SideNavBar from "@components/SideNavBar";
 
 export default function Home() {
   return (
     <div className="flex items-center justify-center h-screen">
       {/* Example Button (ExampleButton.tsx in components folder) */}
-      <ExampleButton buttonText="Add User" />
+      {/* <ExampleButton buttonText="Add User" /> */}
+      <SideNavBar/>
```

Comment:
Can you move the SideNavBar component to src/app/layout.tsx? By putting it in a layout.tsx file it'll ensure that we can see it throughout the app